src/logplex_app.erl

```diff
@@ -29,14 +29,9 @@
 %% Application callbacks
 -export([start/2, start_phase/3, stop/1]).
 
--export([nsync_opts/0
-         ,config/0
-         ,config/1
-         ,config/2
-         ,set_config/2
-         ,start/0
-         ,a_start/2
-        ]).
+-export([nsync_opts/0, priv_dir/0,
+         config/0, config/1, config/2,
+         set_config/2, start/0, a_start/2]).
```

Comment:
Just want to say I'm :-1: on this, as exports are easier to scan top to bottom, but I'm not coding erlang all day...